server/aws/networking.tf

```diff
@@ -105,6 +105,12 @@ resource "aws_vpc_endpoint" "monitoring" {
   subnet_ids = aws_subnet.covidshield_private.*.id
 }
 
+resource "aws_vpc_endpoint" "dynamodb" {
```

Comment:
Is this `dynamodb` endpoint here because the metrics are still deployed in the main `covidshield` VPC, and therefore still need private access?

Reply:
This is attached to this VPC but it can probably use a data lookup and reside in the metrics. However I'm also not opposed to keeping this here until we move the metrics to the other VPC.